cuda_pathfinder/tests/conftest.py

```diff
@@ -9,13 +9,6 @@ def pytest_configure(config):
     config.custom_info = []
 
 
-def pytest_terminal_summary(terminalreporter, exitstatus, config):  # noqa: ARG001
```

Comment:
Sorry I overlooked this somehow:

@rparolin Removing this breaks an important part of the pathfinder tests: the `INFO` lines with paths found are no longer shown. I assume you removed this code for a reason? What was it?

Here is an example of what we are losing by removing this code:

https://github.com/NVIDIA/cuda-python/pull/956#issuecomment-3281597257

Reply:
Are those pathfinder tests not run as part of gated commit checks?  It shouldn't have let me commit if that was the case.  

I think I removed it based on review feedback -- let me double check.